src/components/LineButton/LineButton.js

```diff
@@ -0,0 +1,10 @@
+// @flow
```

Comment:
For maintainability  purposes, is it possible to integrate an additional class into the original Button component. 
```scss
.button {
  all: unset;
  font: inherit;
  cursor: pointer;
  color: inherit;
  padding: 0.4em;
  display: flex;
  align-items: center;
  justify-content: space-around;

  &.underlined {
    padding: 1px 1px 1px 2em;
    border-bottom: 2px solid....
.....
  }
....
```